// @paraboly/react-native-card/build/dist/components/Card/Card.style.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports._container = void 0;
const react_native_1 = require("react-native");
const { width: ScreenWidth } = react_native_1.Dimensions.get("window");
exports._container = (containerHeight, iconDisable, borderRadius, backgroundColor) => ({
    backgroundColor,
    height: containerHeight,
    width: ScreenWidth * 0.92,
    borderRadius: borderRadius || 15,
    overflow: iconDisable ? "visible" : "hidden",
});
exports.default = react_native_1.StyleSheet.create({
    contentContainer: {
        margin: 16,
    },
    bottomRightContainer: {
        bottom: -5,
        opacity: 1,
        position: "absolute",
        alignSelf: "flex-end",
    },
    topRightContainer: {
        top: 0,
        opacity: 1,
        position: "absolute",
        alignSelf: "flex-end",
        justifyContent: "flex-end",
    },
    bottomRightTextStyle: {
        fontSize: 20,
        lineHeight: 22,
        letterSpacing: 0,
        color: "#505e80",
        textAlign: "right",
        alignContent: "flex-end",
    },
    topRightTextStyle: {
        fontSize: 14,
        lineHeight: 18,
        letterSpacing: 0,
        color: "#505e80",
        textAlign: "right",
        alignContent: "flex-end",
    },
    shadowStyle: {
        shadowRadius: 8,
        shadowOpacity: 0.2,
        shadowColor: "#757575",
        shadowOffset: {
            width: 0,
            height: 3,
        },
    },
});
